packages/integration-sdk-runtime/src/synchronization/index.ts

```diff
@@ -275,11 +278,16 @@ export async function uploadDataChunk<
       delay: 200,
       factor: 1.05,
       handleError(err, attemptContext) {
-        if (err.code === 'RequestEntityTooLargeException') {
-          // No reason to retry these errors as the request size ain't gonna change.
-          throw err;
-        }
+        // Did err.code ever work?  I see it as undefined if I console.log its
```

Comment:
Were you able to get a solid handle on this?

Reply:
Got some solid feedback from @austinkelleher on this one.  The difference is due to my testing being running `yarn j1-integration sync` locally versus this running in an ECS container in AWS and making direct calls.  I added in a helper function that checks both so that it will work in both instances but also be a bit more readable.